R/layouts.R

```diff
@@ -251,3 +251,27 @@ split_layout <- function(..., cell_widths = NULL, cell_args = "", style = NULL){
 splitLayout <- function(..., cellWidths = NULL, cellArgs = "", style = NULL) {
   split_layout(..., cell_widths = cellWidths, cell_args = cellArgs, style = style)
 }
+
+#' @export
+#' @rdname flow_layout
+flow_layout <- function(..., cell_args = list(), item_width = "208px", column_gap = "12px", row_gap = "0px") {
+  container_style <- glue::glue(
+    "display: grid;",
+    "grid-template-columns: repeat(auto-fill, {item_width});",
+    "column-gap: {column_gap};",
+    "row-gap: {row_gap};"
+  )
+  item_style <- "align-self: start;"
+  args <- splitArgs(...)
```

Comment:
in utlis.R function name is split_args - not camelCase as here

Reply:
Thanks, done!